docs/style_guide/definitions.md

```diff
@@ -0,0 +1,103 @@
+# Definitions (Top level definitions)
+
+This section will focus on styling the top level definitions available in the Ballerina.
+
+## Indentation
+* No indentation applied for the top level constructs.
+
+```ballerina
+import ballerina/http;
+
+function func1() {
+    int a = 0;
+}
+```
+
+## Imports
+
+* `import` keyword followed by a single space.
+* No spaces between org name , divider(`/`) and package name.
+
+```ballerina
+import ballerina/http;
+```
+* Identifiers such as version and alias is fronted with single space.
+
+```ballerina
+import wso2/twitter version 0.9.0 as twitter;
+
+import abc/foo.bar version 0.1 as foo;
+```
+
+## Function
+* No indentation for function except when defined in an object definition.
+* No space between function name and the open parentheses `(` of function signature.
+```ballerina
+function func1()
+```
+* Closing brace `}` of the function should indent according to the function.
+* If function body has at least on statement, closing brace `}` should fronted with a new line
+and indent accordingly.
+```ballerina
+function func1() {
+    int a = 0;
+}
+```
+* First parameter should not fronted with a space. 
+* `,` comma which separate the parameters followed by a single space.
+* No spaces between last parameter and the closing parentheses.
+
+```ballerina
+
+function func1(int param1, string param2) {}
+
+```
+ 
+* If function is an object bound function there is no space in front and back of the `.`
```

Comment:
shall we use _**before and after**_ instead of _**in front and back of the**_

Reply:
Better use _**object attached functions**_?

Reply:
will change